src/sections/Projects/Project-grid/index.js

```diff
@@ -121,6 +121,13 @@ const ProjectPage = () => {
                 </Col>
               </div>
             </Link>
+            <Link to="/cloud-native-management/playground"
+              className="project__card eleven">
+              <div className="project__card-container">
```

Comment:
Please have a look at **className** since the convention used in other cards is slightly different.